<ui version="4.0" >
 <class>RVMainWindow</class>
 <widget class="QMainWindow" name="RVMainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SequenceTree - Results Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>45</y>
     <width>602</width>
     <height>253</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <property name="spacing" >
     <number>0</number>
    </property>
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>2</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth" >
        <number>3</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="RVFileView" native="1" name="file_view" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget" >
       <widget class="QWidget" name="tab" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>479</width>
          <height>224</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Tab 1</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>479</width>
          <height>224</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpenExperiment" />
    <addaction name="actionSaveExperiment" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actionConfiguration" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuTools" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>298</y>
     <width>602</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>21</y>
     <width>602</width>
     <height>24</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionReconstruction" />
  </widget>
  <action name="actionOpenExperiment" >
   <property name="text" >
    <string>Open Experiment...</string>
   </property>
  </action>
  <action name="actionSaveExperiment" >
   <property name="text" >
    <string>Save Experiment...</string>
   </property>
  </action>
  <action name="actionConfiguration" >
   <property name="text" >
    <string>Configuration...</string>
   </property>
  </action>
  <action name="actionReconstruction" >
   <property name="text" >
    <string>Reconstruction</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RVFileView</class>
   <extends>QWidget</extends>
   <header>rvfileview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
